htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/TextInput.java

```diff
@@ -37,6 +37,15 @@ public void sendKeys(CharSequence... keys) {
     }
 
     /**
+     * Sets this text input's value.
+     *
+     * @param newValue New value for this text input.
+     */
+    public void setText(String newValue) {
+        getWrappedElement().sendKeys(getClearCharSequence() + newValue);
```

Comment:
That would break backward compatibility and would be counterintuitive.
Also there may be situations when you don't want to clear anything, you just want to press a key.